src/mono/mono/mini/interp/interp-simd.c

```diff
@@ -0,0 +1,566 @@
+
+#include "interp-internals.h"
+#include "interp-simd.h"
+
+#ifdef INTERP_ENABLE_SIMD
+
+typedef gint64 v128_i8 __attribute__ ((vector_size (SIZEOF_V128)));
```

Comment:
I'm not sure this would compile in MSVC. But that's probably not a big issue.

Reply:
`INTERP_ENABLE_SIMD` is disabled on msvc due to this reason